packages/components/src/Disclosure/Disclosure.css

```diff
@@ -42,10 +42,12 @@
 .summaryWrap {
   align-items: baseline;
   display: flex;
+  justify-content: space-between;
 }
 
-.summaryWrap h4 {
-  flex: 1 0 0;
```

Comment:
curious about this removal. do we not need this anymore?

I'm not entirely sure why we wanted an inline element to fill out the full width available to it in the first place if all it is is some text, but yeah this would technically be behaving different from `master`

Reply:
Yeah, this is where I'm a bit concern as well, If we keep it, we'll need to remove this behavior when using a customTitle, because I may want a h4 that doesn't follow this flex config.

I figured that having space between will look the same, but I agree with you, it behaves differently from `master`.

Reply:
Do you have any suggestion? What's the default behavior for flex when isn't set?

Maybe I just overwrite the flex behavior in the customSummaryWrap like:

```css
customSummaryWrap h4 {
  flex: default?
}
```

Reply:
ahh right. if someone passes in custom things with their own flex/layout and it happens to contain an h4 - then this rule would interfere with it in a non obvious, and kind of annoying way since it's an internal style.

so while it definitely behaves differently from `master` I'm trying to decide if that even matters....

<img width="776" alt="image" src="https://github.com/user-attachments/assets/67d95710-6eb1-425d-afe3-955856199f2f">

I took a look at the original PR and couldn't find any comments or explanation as to why we wanted the text to grow

I'm struggling to think of why you'd want your element to fill the full width if it didn't need it. since it's just text, it's going to keep looking like empty space either way.... and it's not going to interfere with clicks or anything

I'll see if anyone on the team remembers

Reply:
ahhh ok I get it now

on master if you remove those rules and don't do anything else at all. you get this
<img width="834" alt="image" src="https://github.com/user-attachments/assets/3a9529e9-786f-4389-a934-3548e0f14b11">

which is not desirable. it should be spanning the entire width. so that's why we needed it.

I don't think it's doing the same thing on this PR though. I swear it was still filling the whole width even with only a string, and a small amount of text.

Reply:
@demoraesgui we're good. 

the `h4` selector rule is no longer needed. `space-between` does the same thing in a better way!

Reply:
> ahh right. if someone passes in custom things with their own flex/layout and it happens to contain an h4 - then this rule would interfere with it in a non obvious, and kind of annoying way since it's an internal style.

Yeah this is pretty much what I was trying to avoid.

> @demoraesgui we're good.

Awesome, thanks for the feedback :)